src/libmain.c

```diff
@@ -406,7 +406,7 @@ static gint get_windows_socket_port(void)
 	if (! g_file_test(configfile, G_FILE_TEST_IS_REGULAR))
 	{
 		geany_debug(
-			"No user config file found, use default TCP port (%s).",
+			"No user config file found, use default TCP port (%d).",
```

Comment:
Minor English improvement s/use/using/

Reply:
Thanks, I went a bit further and made the text more similar to the other message text below. Still OK?

Reply:
Ok.